public/angular/js/controllers/reports/show.js

```diff
@@ -1,141 +1,205 @@
 angular.module('Aggie')
 
-.controller('ReportsShowController', [
+  .controller('ReportsShowController', [
```

Comment:
Your IDE may have done this automatically, but everything else is unindented here so please undo this to keep things consistent.